svgsynoptic2/synopticwidget.py

```diff
@@ -121,41 +118,19 @@ def addBackend():
         # some ugly magic to get the path to the SVG file right. It
         # needs to be absolute because local paths go to the base URL.
         abspath = os.path.dirname(os.path.abspath(html))
-        print("absolute path "+abspath + '/' + html)
+        # print(f"absolute path {abspath}/{html}")
 
         with open(html) as f:
             text = f.read().replace("${path}", abspath)  # TODO: use template
             if self._modelNames is not None:
                 if "/*configplaceholder*/" in text:
-                    print("placeholder found")
-                    texta,textb = text.split("/*configplaceholder*/", 1)
-                    text =texta + self._modelNames + textb
+                    # print("placeholder found")
+                    texta, textb = text.split("/*configplaceholder*/", 1)
+                    text = texta + self._modelNames + textb
             view.setHtml(text, base_url)
 
         return view
 
-    def _setup_inspector(self, view):
```

Comment:
Can the Inspector be opened in another way, or has the functionality been removed in pyqt5?

Reply:
As far as I know there is no equivalent QWebInspector for QWebEngine. QWebEngine replaced QtWebKit in PyQt5.